tools/goctl/api/parser/g4/ast/apiparser.go

```diff
@@ -113,7 +119,27 @@ func (p *Parser) parse(filename, content string) (*Api, error) {
 
 	var apiAstList []*Api
 	apiAstList = append(apiAstList, root)
-	for _, imp := range root.Import {
+	p.storeVerificationInfo(root)
+	impApiAstList, err := p.invokeImportedApi(root.Import)
```

Comment:
It is possible that a circular import error will occur, maybe we should fix it in this pr or next pr.

Reply:
I'll fix it.